### ISSUE TITLE
writing imputed variants to stdout

### ISSUE THREAD

**Issue by kermany** [statgen/Minimac4#22]
Hi
Is there an option to write output to `stdout` instead of saving to disk?

**Comment by jonathonl** [comment]
No, there are multiple output files and the output prefix is currently the only way to specify the output paths for these files.
